Source/WebKit/UIProcess/ProcessThrottler.cpp

```diff
@@ -42,7 +42,8 @@
 namespace WebKit {
     
 static const Seconds processSuspensionTimeout { 20_s };
-static const Seconds removeAllAssertionsTimeout { 8_min };
+static const Seconds removeAllAssertionsTimeout { 1_min };
+static const Seconds temporaryForegroundAssertionTimerTimeout { 8_min };
```

Comment:
Temporary is not a clear enough name. We want a name that explains what this is for.